tests/notebooks/generate_openl3_regression_data.ipynb

```diff
@@ -382,7 +718,7 @@
    "name": "python",
    "nbconvert_exporter": "python",
    "pygments_lexer": "ipython3",
-   "version": "3.6.4"
+   "version": "3.6.7"
   }
  },
```

Comment:
Other than convenience, is there a strong motivation for generating the regression data in a notebook as opposed to a `.py` script? If not, would it not be preferable to transfer this code into a script?

Reply:
There's not a strong motivation for keeping the regression data in a notebook. I can move it to a script.